app/_de_tools/server.md

```diff
@@ -1,18 +1,23 @@
 ---
 number: 11
-header: "Have your server at home"
-time: "10h"
-skills: 5
-spy_reduction: 2
-incomfort: 4
+header: "Einen eigenen Server zu Hause haben"
+time: "8h"
+skills: 3
+spy_reduction: 4
+incomfort: 3
```

Comment:
It is difficult also, let's discuss about asymetric bandwitdh, laws (it is sometimes illegal to host from home), electricty outage, Police entering your place, you being kicked out from your house from your boy/girl friend?

Reply:
In fact, it's no more illegal. Numericable, in France, needed to remove this line from its CGU because it was too much. OVH, for example, my ISP, allows selfhosting. In Belgique, many users can selfhost and even in Turkey, they are installing and developing communities. For the USA and America in general, I don't know. For the moment there are many problem about that in the USA. 

The main problem is email selfhosting I think because most of the ISP (without OVH and FFDN) close the port 25 or allows it but the email are always send as SPAM.

(a bit France-oriented ...)